### ISSUE TITLE
Issues with plugin on Windows/WSL

### ISSUE THREAD

**Issue by petewilcock** [aquasecurity/vscode-tfsec#26]
**Describe the bug**

**Desktop (please complete the following information):**
 - OS: Windows 
 - Tfsec Version [e.g. v0.62.0]: v1.16.2
 - VSCode Version: 1.66.0

**Additional context**

When I open my workspace in WSL with TFSec installed, the Findings Explorer Populates results, however when I attempt to click on the individual finding, it pops up saying 'File not found'. Seems to be an issue with pathing, because the path indicated in the explorer is not the correct path to the file, but seems to be a relative path from within Windows AppData:

![image](https://user-images.githubusercontent.com/6526265/161958205-2f35e5f3-8270-4292-8c09-942844f508fe.png)

If I open the workspace in Windows, and install tfsec in the Windows Shell and point to it on PATH, tfsec runs and the Output says 'Reloading the Findings Explorer content', but it never updates. The command run looks like this:

```
Running tfsec to update results
command: --force-all-dirs,--exclude-downloaded-modules,--verbose,--soft-fail,--format=json,--out=c:\Users\petew\AppData\Roaming\Code\User\workspaceStorage\7bbdd34ce22445141ad71cd11c88122a\tfsec.tfsec\.tfsec\9677760a-5f3d-4645-ad89-c781984a2fc8_results.json,a:\Google Drive\code\projects\serverless-wordpress
1 file(s) written: c:\Users\petew\AppData\Roaming\Code\User\workspaceStorage\7bbdd34ce22445141ad71cd11c88122a\tfsec.tfsec\.tfsec\9677760a-5f3d-4645-ad89-c781984a2fc8_results.json
```
So whichever way I try to run it, I can't get any usage out of it :(

**Comment by owenrumney** [comment]
Hey @petewilcock - 1.16.2 got pulled because it was broken. Can you retry with 1.17.0?

**Comment by petewilcock** [comment]
Thanks @owenrumney I see results in Windows now, however the pathing issue still exists where it's looking for the files in my AppData directory rather than the actual path to file.

**Comment by owenrumney** [comment]
Hey @petewilcock  - apologies for the delay - this should be resolved in v1.9.0 of the vscode plugin

**Comment by petewilcock** [comment]
@owenrumney Confirmed fixed in 1.9.0. Thanks! 🥇
